src/code/middleware/its-log.js

```diff
@@ -31,19 +31,19 @@ export function initializeITSSocket(guideServer, socketPath, store) {
 
     if (event.isMatch("ITS", "HINT", "USER")) {
       // TODO: Re-enable dispatch once ITS is fixed
```

Comment:
These TODO messages aren't quite right at this point, but things are sufficiently fluid that it probably doesn't matter much.